Sources/Views/MusicProgressSliderCell.m

```diff
@@ -0,0 +1,83 @@
+//
+//  MusicProgressSlider.m
+//  Hermes
+//
+//  Created by Xinhong LIU on 19/4/15.
+//
+//
+
+#import "MusicProgressSliderCell.h"
+
+@implementation MusicProgressSliderCell
+
+- (NSRect)knobRectFlipped:(BOOL)flipped {
+  CGPoint sliderOrigin = [self barRectFlipped:false].origin;
+  CGSize knobSize = CGSizeMake(2.0, 8.0); // this value is measured in iTunes App
+  CGSize sliderSize = [self barRectFlipped:false].size;
+  
+  CGPoint knobOrigin;
+  knobOrigin.x = truncf(sliderOrigin.x + [self progressInPercentage]
+                        * (sliderSize.width - knobSize.width));
+  // truncf is important to make knob's border clear and sharp.
+  knobOrigin.y = sliderOrigin.y + sliderSize.height - knobSize.height;
+  
+  NSRect knobRect = NSMakeRect(knobOrigin.x, knobOrigin.y, knobSize.width, knobSize.height);
+  return knobRect;
+}
+
+- (CGFloat) progressInPercentage {
```

Comment:
As per the [Documentation/Contributing.md](https://github.com/HermesApp/Hermes/blob/master/Documentation/Contributing.md) document, please remove the space after the return type. (It's extra whitespace) -- it should read like `- (CGFloat)progressInPercentage {`, not `- (CGFloat) progressInPercentage {`. (See the single space removed between `(CGFloat)` and `progressInPercentage`.)

Please make this change throughout this file.